test/e2e/commands/project/test_propertyclass.py

```diff
@@ -1,118 +1,54 @@
-
-        self.assertIsNotNone(self.onto.iri)
-        self.last_modification_date = self.onto.lastModificationDate
-
-    def tearDown(self) -> None:
-        """
-        Logs out from DSP-API and remove test ontology.
-        For each test method, a new TestCase instance is created, so tearDown() is executed after each test method.
-        """
-        result = self.onto.delete()
-        self.assertIsNotNone(result)
-        self.con.logout()
-
-    def test_PropertyClass_create(self) -> None:
-        """
-        create new property class
-        """
-        self.last_modification_date, property_class = PropertyClass(
-            con=self.con,
-            context=self.onto.context,
-            name=self.name,
-            ontology_id=self.onto.iri,
-            rdf_object=self.rdf_object,
-            label=self.label,
-            comment=self.comment,
-        ).create(self.last_modification_date)
-
-        self.onto.lastModificationDate = self.last_modification_date
-
-        self.assertIsNotNone(property_class.iri)
-        self.assertEqual(property_class.name, self.name)
-        self.assertEqual(property_class.label["de"], self.label["de"])
-        self.assertEqual(property_class.comment["de"], self.comment["de"])
-
-        # get ontology data
-        self.onto = self.onto.read()
-        self.last_modification_date = self.onto.lastModificationDate
-        self.last_modification_date = property_class.delete(self.last_modification_date)
-
-        # get ontology data
-        self.onto = self.onto.read()
-
-    def test_PropertyClass_update(self) -> None:
-        self.onto = self.onto.read()
-
-        # create test resource class
-        self.last_modification_date, property_class = PropertyClass(
-            con=self.con,
-            context=self.onto.context,
-            name=self.name,
-            ontology_id=self.onto.iri,
-            rdf_object=self.rdf_object,
-            label=self.label,
-            comment=self.comment,
-        ).create(self.last_modification_date)
-        self.onto.lastModificationDate = self.last_modification_date
-        self.assertIsNotNone(property_class.iri)
-
-        # modify the property class
-        property_class.addLabel("en", "This is english comment")
-        property_class.rmLabel("de")
-        property_class.addComment("it", "Commentario italiano")
-        self.last_modification_date, property_class_updated = property_class.update(self.last_modification_date)
-        self.onto.lastModificationDate = self.last_modification_date
-        self.assertEqual(property_class_updated.label["en"], "This is english comment")
-        self.assertEqual(property_class_updated.comment["it"], "Commentario italiano")
 
-        # delete the resource class to clean up
-        self.last_modification_date = property_class_updated.delete(self.last_modification_date)
-        self.onto.lastModificationDate = self.last_modification_date
+def test_PropertyClass_create() -> None:
```

Comment:
My intention was to refactor this test, so that it doesn't need `Ontology.delete()` any more. Ideally, it should still test the same behaviours, just in a simpler way. You are welcome to double-check that it still tests what it tested before.